dsgrid/__init__.py

```diff
@@ -1,2 +1,2 @@
-from ._version import __version__, __author__, __copyright__
-from dsgrid.utils.timing import timer_stats_collector
+from dsgrid.utils.timing import timer_stats_collector  # noqa: F401
+from ._version import __version__, __author__, __copyright__  # noqa: F401
```

Comment:
why # noqa on these imports?

Reply:
flake8 complains because they are unused in this file. We use them elsewhere, as is `dsgrid.__version__`. So, disable the warning.